plugins/login/fps_login/static/index.html

```diff
@@ -0,0 +1,123 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+
+<body class="" dir="ltr">
+
+  <noscript>
+    <div id='noscript'>
+      Jupyverse requires JavaScript.<br>
+      Please enable it to proceed.
+    </div>
+  </noscript>
+
+  <div id="header" role="navigation" aria-label="Top Menu">
+    <div id="header-container" class="container">
+			<div class="navbar-header">
+				<div id="jupyter_server" class="nav navbar-brand navbar-left">
+					<a href="/" title='Jupyverse'>
+						<img src='static/login_page/logo/logo.png' alt='Jupyverse' />
+					</a>
+				</div>
+			</div>
+			<div class="nav navbar-nav">
+				<div id="jupyter_server" class="nav navbar-brand navbar-right">
+					<button id="github_btn" class="github">
+						<img src='static/login_page/logo/github.svg' alt='GitHub' />
+					</button>
+				</div>
+			</div>
+    </div>
+    <div class="header-bar"></div>
+  </div>
+
+  <div id="site">
+		<div id="jupyter-main-app" class="container">
+			<div class="row">
+				<div class="navbar col-sm-8">
+					<div class="navbar-inner">
+						
+						<form action="/auth/login" method="post" autocomplete="on">
+							<div class="container">
+								<div class="center-nav">
+									<div class="navbar-form pull-left">
+										<label for="uname"><strong>Email:</strong></label>
+										<input id="email_input" type="text" placeholder="Enter email" name="username" autocomplete="on" class="form-control" required>
+									</div>
+								</div>
+							</div>
+							<div class="container">
+								<div class="center-nav">
+									<div class="navbar-form pull-left">
+										<label for="psw"><strong>Password:</strong></label>
+										<input id="psw_input" type="password" placeholder="Enter Password" name="password" autocomplete="on" class="form-control" required>
+									</div>
+								</div>
+							</div>
+							<div class="container">
+								<div class="center-nav">
+									<div class="navbar-form pull-left">
+										<button id="user_btn" type="submit" class="btn btn-default">Login</button>
+									</div>
+								</div>
+							</div>
+						</form>
+
+						<div class="container">
+								<div class="center-nav">
+									<div class="navbar-form pull-left">
+										<label for="password_input"><strong>Token:</strong></label>
+										<input id="token_input" type="password" name="password" class="form-control">
+										<button id="token_btn" type="submit" class="btn btn-default">Login</button>
+									</div>
+								</div>
+						</div>
+
+						<div class="container">
+							<div class="center-nav">
+								<div class="navbar-form pull-left">
+									<p>To login in JupyterLab you can use one of the following options:</p>
+									<ul>
+										<li>Click the GitHub button to login using your GitHub account</li>
+										<li>Introduce a login token</li>
+										<li>Or login as an <a id="anonymous_btn" href="/lab">anonymous</a> user</li>
```

Comment:
Clicking on the anonymous link has no effect. Also, the login/password registration is not mentioned.